src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/servicebus/model/PaymentMessage.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmcts.reform.bulkscan.payment.processor.service.servicebus.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class PaymentMessage {
+
+    public final String envelopeId;
+    public final String ccdCaseNumber;
+    public final boolean isExceptionRecord;
+    public final String poBox;
+    public final String jurisdiction;
+    public final List<PaymentInfo> payments;
+
+
+    public PaymentMessage(
+        @JsonProperty(value = "envelope_id", required = true) String envelopeId,
+        @JsonProperty(value = "ccd_case_number", required = true) String ccdCaseNumber,
+        @JsonProperty(value = "is_exception_record", required = true) boolean isExceptionRecord,
+        @JsonProperty(value = "po_box", required = true) String poBox,
+        @JsonProperty(value = "jurisdiction", required = true) String jurisdiction,
+        @JsonProperty(value = "payments", required = true) List<PaymentInfo> payments) {
```

Comment:
is `site_id` missing? it's there in the jira ticket

Reply:
we will map jurisdiction  to site id .there is a site SiteConfiguration class which reads these stite id comnfigs.

Reply:
You should be mapping pobox or container to siteid and not jurisdiction

Reply:
I will use po box map to site_id.